src/styles/sidebar/components/PaginationNavigation.scss

```diff
@@ -5,22 +5,24 @@
 
 .PaginationNavigation {
   @include utils.font--large;
-  @include layout.row;
 
-  &__relative {
-    // Set a minimum width for the previous and next button containers so that
-    // there is no jumping around as they appear and disappear.
-    min-width: 5em;
+  display: grid;
+  grid-template-columns: 8em auto 8em;
+  align-items: center;
+  grid-template-areas: 'previousPage numberedPages nextPage';
```

Comment:
Perhaps worth adding a comment here about the intended layout behavior from the user's point of view, to ensure that is preserved in case this is ever changed for any reason.

Reply:
I added a fairly verbose comment.